lib/project_types/script/messages/messages.rb

```diff
@@ -144,6 +144,9 @@ module Messages
 
           script_upload_cause: "Fail to upload script.",
           script_upload_help: "Try again.",
+
+          api_library_not_found_cause: "Failed to find API Library in dependencies.",
+          api_library_not_found_help: "Try re-installing project dependencies.",
         },
```

Comment:
Based on the code, it appears that this error happens if the Script API package is not found.

When we create the project, we add the appropriate Script API package to the `package.json` file. If it is missing, it could be because the developer deleted their `node_modules`. In that case, they should do `npm install` or `yarn` to install dependencies again.

Another possibility is that the developer manually removed the Script API package from `package.json`. I'm not sure why anyone would do that, but in that case, they can re-add it by running `npm install {packageName}` where `{packageName}` is the API package name.